### PR TITLE
Enable the use of torch in graph-env with examples

### PR DESCRIPTION
**Pull request by davebiagioni**
Implement torch functionality and demonstrate on graph-env examples using PPO and AZ from contrib algorithms

### CODE REVIEW

File: setup.cfg [hunk 1]
```diff
@@ -15,6 +15,9 @@ install_requires =
   tensorflow
   networkx
   ray[tune,rllib]
+  torch
+  torchaudio
+  torchvision
```
Comment (pstjohn) [inline setup.cfg:20]: Do we need these? Just wondering what's provided here versus the base package

Comment (davebiagioni) [reply setup.cfg:20]: Hmm, good question, probably not.  I do this by default because that's what they have in their docs.

**Comment by pstjohn** [comment]
OK, this is looking a lot better. I think there were some issues and strange gotchas in the TorchHallwayModel, and a couple lingering oddities in the torch implementations for the graph_model utilities. 

I wasn't able to get DQN working for torch, I think we want to inherit from `DQNTorchModel`, but the MRO there still seems off.

If you had a chance we could probably try alphazero for the Hallway example and make sure there aren't any remaining barriers other than getting torch policy models in place.

**Review by pstjohn — APPROVED** [review]

File: setup.cfg (hunk 1 above)
Comment (davebiagioni) [reply setup.cfg:20]: great

**Pull request merged**